### PR TITLE
Add missing User API token

### PR DESCRIPTION
**Pull request by fridex**
## This introduces a breaking change

- [x] No

## This Pull Request implements

To enable authenticated workflow, we need to provide a token to User API. See:

https://github.com/thoth-station/user-api/blob/20ac6db4154e5480970727ff831d0272bfbd5d1c/thoth/user_api/configuration.py#L37

### CODE REVIEW

File: user-api/base/deploymentconfig.yaml [hunk 1]
```diff
@@ -38,6 +38,11 @@ spec:
               value: "khemenu.thoth-station.ninja"
             - name: "THOTH_SENTRY_IGNORE_EXCEPTION"
               value: "thoth.user_api.exceptions.NotFoundException,thoth.storages.exceptions.NotFoundError"
+            - name: THOTH_USER_API_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  key: token
+                  name: thoth
```
Comment (fridex) [inline user-api/base/deploymentconfig.yaml:45]: This does not exist. Not sure if it should be mapped to another key. CC @harshad16

Comment (harshad16) [reply user-api/base/deploymentconfig.yaml:45]: please use the key `thoth-secret` for this.

Comment (fridex) [reply user-api/base/deploymentconfig.yaml:45]: Adjusted, PTAL

**Review by harshad16 — APPROVED** [review]
lgtm , thank you

**Pull request merged**